migrations/07_add_token_info_to_balances.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE balances
+ADD symbol VARCHAR;
+
+ALTER TABLE balances
+ADD decimals INTEGER NOT NULL DEFAULT 0;
```

Comment:
I think this should be a separate table. otherwise we'll have to duplicate this information all over